### ISSUE TITLE
Group search field not correctly displayed

### ISSUE THREAD

**Issue by fcharlaix-opendsi** [nextcloud/announcementcenter#607]
### Steps to reproduce
1. Open the app
2. Click on the (...) button
3. Click on the groups search field

### Expected behaviour
A list of all groups name should appear to select with one is allowed to see the announcement.

### Actual behaviour
A list of `[object Object]` is displayed, when selected its show as : `{"id": "***GROUP ID***","label": "***GROUP LABEL***"}`.

### Server configuration

**Operating system**: Debian 11, Docker

**Web server:** Apache 2, Docker (Apache 2)

**Database:** MariaDB 10.3.36, SQLite 3

**PHP version:** 7.4.33, 8.1.14

**Nextcloud version:** 25.0.2, 25.0.3

**Where did you install Nextcloud from:** Official ZIP, Docker

**Signing status:**

```
No errors have been found.
```
```
No errors have been found.
```

**List of activated apps:**

```
Enabled:
  - activity: 2.17.0
  - announcementcenter: 6.5.0
  - approval: 1.0.12
  - circles: 25.0.0
  - cloud_federation_api: 1.8.0
  - comments: 1.15.0
  - contactsinteraction: 1.6.0
  - dashboard: 7.5.0
  - dav: 1.24.0
  - federatedfilesharing: 1.15.0
  - federation: 1.15.0
  - files: 1.20.1
  - files_accesscontrol: 1.15.1
  - files_automatedtagging: 1.15.0
  - files_pdfviewer: 2.6.0
  - files_rightclick: 1.4.0
  - files_sharing: 1.17.0
  - files_trashbin: 1.15.0
  - files_versions: 1.18.0
  - firstrunwizard: 2.14.0
  - flow_notifications: 1.5.0
  - groupfolders: 13.1.1
  - impersonate: 1.11.0
  - logreader: 2.10.0
  - lookup_server_connector: 1.13.0
  - nextcloud_announcements: 1.14.0
  - notifications: 2.13.1
  - oauth2: 1.13.0
  - password_policy: 1.15.0
  - photos: 2.0.1
  - privacy: 1.9.0
  - provisioning_api: 1.15.0
  - recommendations: 1.4.0
  - related_resources: 1.0.3
  - serverinfo: 1.15.0
  - settings: 1.7.0
  - sharebymail: 1.15.0
  - support: 1.8.0
  - survey_client: 1.13.0
  - systemtags: 1.15.0
  - text: 3.6.0
  - theming: 2.0.1
  - twofactor_backupcodes: 1.14.0
  - updatenotification: 1.15.0
  - user_status: 1.5.0
  - viewer: 1.9.0
  - weather_status: 1.5.0
  - workflow_pdf_converter: 1.10.0
  - workflowengine: 2.7.0
Disabled:
  - admin_audit
  - bruteforcesettings
  - encryption
  - files_external
  - suspicious_login
  - twofactor_totp
  - user_ldap
```

**Nextcloud configuration:**

```
{
    "system": {
        "htaccess.RewriteBase": "\/",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "apps_paths": [
            {
                "path": "\/var\/www\/html\/apps",
                "url": "\/apps",
                "writable": false
            },
            {
                "path": "\/var\/www\/html\/custom_apps",
                "url": "\/custom_apps",
                "writable": true
            }
        ],
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "localhost:8080"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "sqlite3",
        "version": "25.0.3.2",
        "overwrite.cli.url": "http:\/\/localhost:8080",
        "installed": true,
        "app_install_overwrite": [
            "impersonate"
        ]
    }
}
```

**Are you using an external user-backend, if yes which one:** LDAP, None

### Client configuration
**Browser:** Vivaldi 5.7.2921.53 (Chrome/110.0.0.0), Firefox 110.0

**Operating system:** Arch Linux, Windows 10

#### Nextcloud log (data/owncloud.log)
```
Nothing to see here
```

#### Browser log
JS :
```
No translation for appId "announcementcenter" have been registered
```

**Comment by nickvergessen** [comment]
Which version of the app are you running?

**Comment by fcharlaix-opendsi** [comment]
Both Nextcloud instances where I have reproduced the bug use the version `6.5.0` of the app.

**Comment by pascald0210** [comment]
Same issue after upgrading to 6.5.0
NC 25.03

**Comment by nickvergessen** [comment]
Reported and fixed upstream:
https://github.com/nextcloud/nextcloud-vue/pull/3795

**Comment by NL-CCMLM** [comment]
Same issue for us
Announcementcenter app v 6.5.0
NC v 25.0.3

**Comment by nickvergessen** [comment]
Fixed with https://github.com/nextcloud/announcementcenter/pull/611

![image](https://user-images.githubusercontent.com/213943/221837240-3ffe6aa0-c971-424c-8a66-6f7afdc2042a.png)


Will release later this week

**Comment by nickvergessen** [comment]
Released 6.5.1

**Comment by thomasosche** [comment]
The update works perfectly. Many Thanks!
